ops/charm.py

```diff
@@ -893,7 +893,68 @@ class ContainerMeta:
 
     Attributes:
         name: Name of container (key in the YAML)
+        mounts: :class:`ContainerStorageMeta` mounts available to the container
```

Comment:
At what point in the charm lifecycle is `ContainerStorageMeta` populated ? For example is it populated only after the `StorageAttachedEvent`. If so, should this be made explicit in the doc string.

Reply:
`ContainerStorageMeta` is populated when the charm metadata is -- i.e. at charm initialization, like the rest of the `*Meta` classes. No dependency on events.